content/en/developers/metrics/metrics_type.md

```diff
@@ -0,0 +1,167 @@
+
+In the Datadog web application there are five metric types (though one is deprecated):
+
+* COUNT
+* COUNTER (deprecated)
+* GAUGE
+* RATE
+* DISTRIBUTION
+
+A metric's type is stored as metrics metadata and is used to determine how a metric is interpreted throughout the application by determining default time aggregation function and `as_rate()`/`as_count()` behavior. The `as_count()` and `as_rate()` modifiers behave differently for different web application metric types.
+
+## Metric type definition.
+
+To better understand the different metrics types, what they represent and how to manipulate them within Datadog let's start with an example:
+
+You have two web servers `web_1` and `web_2`, each one of them is dealing with HTTP requests over time.
+
+They receive both:
+
+  * 1 request per second for 10 seconds
+  * 2 requests per second for 10 seconds
+  * 0 request for 10 seconds
+
+and so on and so fourth.
+
+{{< tabs >}}
+{{% tab "Count" %}}
+
+**Metrics typed with `Count` are used to count things over a period of time.**
+
+For instance let's say the `number.of.requests` metrics is reported every 10 seconds to Datadog with the `count` type for `web_1`.
+
+The metric has then the following shape: 10, then 20 then 0
+
+Each data point of this metrics represent the amount of request received during the 10 second interval.
+
+{{% /tab %}}
+{{% tab "Gauge" %}}
+
+**Gauges measure the value of a particular thing over time. It's a snapshot of a value associated to a timestamp.**
+
+For instance let's say the `number.of.requests` metrics is reported every 10 seconds to Datadog with the `gauge` type for `web_1`.
+
+The metric has then the following shape: 10 , then 30 then 30.
+
+{{% /tab %}}
+{{% tab "Rate" %}}
+
+**Rates represent the derivative of a metric, it's the value variation of a metric on a defined time interval.**
+
+For instance let's say the `number.of.requests` metrics is reported every 10 seconds to Datadog with the `gauge` type for `web_1`.
+
+The metric has then the following shape: 1 , then 2 then 0.
+
+{{% /tab %}}
+
+{{% tab "Distribution" %}}
+
+<div class="alert alert-warning">
+This feature is in beta. <a href="https://docs.datadoghq.com/help/">Contact Datadog support</a> to enable distribution metrics for your account.
+</div>
+
+Distributions are a metric type that aggregate values sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure. Distribution metrics are designed to instrument logical objects, like services, independently from the underlying hosts, and solve the problem created by Agent-level aggregation.
+
+Unlike the histogram metric type that aggregates on the Agent-side, distributions send all raw data collected during the flush interval and aggregations occur server-side. Because the underlying data structure has not been aggregated and represents raw data, distributions provide two major features:
```

Comment:
```suggestion
Unlike the histogram metric type that aggregates on the Agent-side, distributions send all raw data collected during the flush interval, and aggregations occur server-side. Because the underlying data structure has not been aggregated and represents raw data, distributions provide two major features:
```